analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.cs

```diff
@@ -222,12 +231,14 @@ public partial class EventHandlerSample1
     public class PropertyAccess
     {
         private int OnlyRead { get; set; }  // Noncompliant {{Remove the unused private set accessor in property 'OnlyRead'.}}
-//                                  ^^^^
+//                                  ^^^
         private int OnlySet { get; set; }
         private int OnlySet2 { get { return 42; } set { } } // Noncompliant {{Remove the unused private get accessor in property 'OnlySet2'.}}
-//                             ^^^^^^^^^^^^^^^^^^
+//                             ^^^
         private int NotAccessed { get; set; }   // Noncompliant {{Remove the unused private property 'NotAccessed'.}}
-//      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+//                  ^^^^^^^^^^^
+        private int Calculated => 1; // Noncompliant {{Remove the unused private property 'Calculated'.}}
+//                  ^^^^^^^^^^
```

Comment:
I took a quick look and it seems that arrow expressions are missing in general. Could you add
```cs
Prop { get => 1; }
Prop { set => _Prop = value; }
```

Indexer are also missing and most likely are FPs
```cs
private int this[int i] => 1;
private int this[int i] { get => 1; }
private int this[int i] { set => _ = value; }
private int this[int i] { get { return 1; } set { _ = value; } }
```

Reply:
Good catch. I've changed the code so that it only highlights the `this `keyword for unused indexers.
Also added test cases for expression-bodied properties.